docs/superpowers/specs/2026-05-26-room-id-wire-convention-design.html

```
@@ -0,0 +1,638 @@
+</section>
+
+<section id="architecture">
+  <h2>5 · Architecture</h2>
+
+  <div class="mermaid">
+flowchart LR
+    subgraph backend [backend]
+        EVT["socket_events.py<br/><b>RoomScopedEvent</b><br/>room_id: UUID<br/>room_address: str<br/>for_room(room, **kw)"]
+        BC["broadcast.py<br/><b>broadcast_to_room</b><br/>room_channel(uuid)"]
+        EMIT["~25 emit sites<br/>routes/*.py<br/>socketio.py<br/>zndraw_joblib/*.py"]
+        EVT --> EMIT
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)